### PR TITLE
Fixed analysis warning about if statements in PlateHorizontalGraph.

### PR DESCRIPTION
**Pull request by FridaTveit**
Changed empty if statements to while statements. Removed unused argument 'count'. Removed unused field 'handle'. Changed Plate.java to reflect this change. This relates to issue #18 .

### CODE REVIEW

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java [hunk 1]
```diff
@@ -25,46 +25,42 @@
...
+    public Vector<Peak> findPeak() {
         if (PlateHorizontalGraph.horizontalDetectionType == 1) {
-            return this.findPeak_edgedetection(count);
+            return this.findPeak_edgedetection();
         }
-        return this.findPeak_derivative(count);
+        return this.findPeak_derivative();
     }
 
-    public Vector<Peak> findPeak_derivative(int count) {
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Optional: Would you formatting the name of this and the other `findPeak` method to camelCase?

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java [hunk 2]
```diff
@@ -25,46 +25,42 @@
...
-        int a, b;
+    public Vector<Peak> findPeak_derivative() {
+        int a = 2;
+        int b = this.yValues.size() - 1 - 2;
         float maxVal = this.getMaxValue();
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Optional: Remove all the extra `this` statements if you feel like it :) See #42

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Will do!

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java (hunk 2 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Will do!

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Done :)

File: src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java (hunk 2 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java]: Done :)

**Pull request merged**